### ISSUE TITLE
Improvements to the way extra information is given when a install/uninstall actions is done (`WarnPopup` alternarives)

### ISSUE THREAD

**Issue by dalthviz** [napari/napari-plugin-manager#69]
> Ended up pushing a commit making the popup appear besides the installed label counter text, so things are looking something like the following:

Sorry for the bikeshedding here, but in my eyes this pop up in general is a bit weird. It's supposed to give information about a the plugin dialog. This small box looks more like a tooltip that, if anything, should appear next to the button that triggered the action.

I feel that a [message bar](https://acorn.firefox.com/latest/components/message-bar-V5RTUg0H) like this is more appropriate. It would be placed either at the top of the dialog box, or at the bottom. It should cover the full width of the dialog too.

![image](https://github.com/napari/napari-plugin-manager/assets/2559438/d5a22db5-65b6-43bb-adb6-06f040bedab9)

And maybe this is useful for some other notifications. But speaking of that... if this is too much work or you see it as unnecessary... why not push a message to the napari notification system?

_Originally posted by @jaimergp in https://github.com/napari/napari-plugin-manager/issues/68#issuecomment-2220628204_

**Comment by dalthviz** [comment]
Some other related comments: https://github.com/napari/napari-plugin-manager/pull/68#issuecomment-2220717203

**Comment by goanpeca** [comment]
I am not sure I recall the design decision that went into doing this as an extra popup dialog on top of yet another modal dialog.

The napari notification system would be a good option, if the napari plugin manager was a panel or tab inside the main windows. This is something a suggested a [long time ago](https://github.com/napari/napari/pull/2607), since it was not favored at the time, and we continue being an extra dialog on top of the main app, I would suggest we just use the bottomg of the dialog for messaging, since that is close to the show status button where we display fine grained information about the processes.

**Comment by psobolewskiPhD** [comment]
I don't see an issue with the napari notification.
User installs plugin, let's say they don't notice anything, but then closes installer -- there's the notification. As long as it's persistent and doesn't auto-close anyways?

**Comment by psobolewskiPhD** [comment]
Looks like right now all things use 4000 ms. 
I think warnings/errors persisting is reasonable? I'll make a napari PR and we can see what folks think.

**Comment by goanpeca** [comment]
I would actually like to see persistence in terms of having a history. I do not mind warnings hiding after a "reasonable" time (right now it feels too short BTW), if I have a history and I can go there and checked what happened with more detail

<img width="465" alt="Screenshot 2024-07-20 at 1 40 51 PM" src="https://github.com/user-attachments/assets/b69dfaaf-b9ef-4cce-8af7-a00fe74b6829">

**Comment by psobolewskiPhD** [comment]
That would be nice I think, but but if the notification disappears before you know it was there -- because it was obscured by the plugin installer, then how would you know to check?

**Comment by goanpeca** [comment]
Good point!, I guess both things might be needed in the long run :) so that during the napari session, the user can check all the notifications that came about

**Comment by psobolewskiPhD** [comment]
I do like the idea of the "bell" icon and status indicator though!

**Comment by goanpeca** [comment]
Is there an issue on napari @psobolewskiPhD , maybe we could open one in order to make the notifications manager a bit more sophisticated :)

**Comment by jaimergp** [comment]
Gonzalo and I were talking about this and might make more sense to have a more useful "status bar" and rearrange some of the UI elements to make enough space for it. I'll rename this issue accordingly.
